test/utils/mocha.ts

```diff
@@ -1,11 +1,79 @@
+import deepcopy from 'deepcopy';
 import { expect } from 'chai';
 
+import FullStory from '../mocks/fullstory-recording';
 import { MockClass, Call } from '../mocks/mock';
 import { DataLayerDetail } from '../../src/event';
-import { DataLayerObserver, DataLayerConfig } from '../../src/observer';
+import { DataLayerObserver, DataLayerConfig, DataLayerRule } from '../../src/observer';
 import MonitorFactory from '../../src/monitor-factory';
 import { BuiltinOptions, OperatorFactory } from '../../src/factory';
 
+/**
+ * Gets or sets a global object. Use this rather than writing `globalThis as any`.
+ * @param key The key corresponding to the global object (e.g. FS in window['FS']).
+ * @param value Optional object that when provided will set the global's value
+ */
+export function global(key: string, value?: any): any {
```

Comment:
it might be better to be more explicit with `getGlobal` and `setGlobal` -  this way the behavior doesn't change depending on whether a parameter is present.